.gitignore

```diff
@@ -38,3 +38,6 @@ node_modules
 website/resources
 website/public
 website/.hugo_build.lock
+
+# MacOs 
+.DS_Store
```

Comment:
Please also add a linebreak after. Files once committed stay in the tree even if gitignored so you'll have to `git rm` the files as well to get them removed from the commit

Reply:
I did git rm the files though, let me add a line break

Reply:
ah you're right, my bad